api/lib/infrastructure/utils/RedisClient.js

```diff
@@ -59,10 +59,11 @@ class RedisClient {
   }
 
   async quit() {
-    if (this._client.status == 'end') {
+    try {
+      await this._client.quit();
+    } catch (e) {
       return;
     }
```

Comment:
Et si on s'assurait qu'on était dans ce cas-là ?
```suggestion
    } catch (e) {
    if (e.message = 'foo'){
       return;
    } else {
      logger.info(`${e.message} happened during redis client exit `
    }
```

Reply:
Done